lib/network_utils.ml

```diff
@@ -33,6 +33,7 @@ let modprobe = "/sbin/modprobe"
 let ethtool = ref "/sbin/ethtool"
 let bonding_dir = "/proc/net/bonding/"
 let dhcp6c = "/sbin/dhcp6c"
+let fcoedriver = "/opt/xensource/fcoe/fcoe_driver.py"
```

Comment:
It would be good to allow setting this from the xcp-networkd config file. You can follow the way it is done for `ethtool`:
* Make the string above a `ref`.
* Add a section to networkd.ml, like this: https://github.com/xapi-project/xcp-networkd/blob/master/networkd/networkd.ml#L37